test/controllers/offer.controller.test.js

```diff
@@ -155,6 +155,7 @@ describe('Offer Controller', () => {
               expect(res.body.offer).to.have.property('userInfo');
               expect(res.body.offer.vendorInfo._id).to.be.eql(vendorUser._id.toString());
               expect(res.body.offer.propertyInfo._id).to.be.eql(newProperty._id.toString());
+              expect(res.body.offer.expires).to.have.string('22:59:59.999');
```

Comment:
Not currently sure if the converted date format (expires) is the issue but the end of the day should be `23:59:59.999` since we have 24 hours. 

Likely issue is that `expires` is supposed to be in GMT+1 format to correspond with our own date.

https://date-fns.org/v2.21.1/docs/endOfDay
```suggestion
              expect(res.body.offer.expires).to.have.string('23:59:59.999');
```

Reply:
This is actually correct but in a tricky way. The expiry date is set with respect to the user's timezone. So the issue with this is, the server's clock is at GMT and the machine is at GMT+1. So the date returned is correct (for the server) with respect to the user's clock.

Reply:
>  the server's clock is at GMT and the machine is at GMT+1

I am sorry, I have to disagree with this ☝🏽 as the test is run on Travis CI which I believe serves as the server and machine (or whatever you mean by this). 

The returned end-of-day time in the test is `22:59:59.999`, the response doesn't have any reference to the GMT or GMT+1, so I believe this is wrong. Even if it will return `22:59:59.999`, then the GMT should be specified else the time should be in GMT+1 to ensure consistency when it is being viewed from different parts of the world.